metadata-ingestion/tests/integration/powerbi/golden_test_admin_access_not_allowed.json

```diff
@@ -1,34 +1,4 @@
 [
-{
```

Comment:
Could you please make sure existing golden file should not get updated because of your code changes ? description related changes or new PowerBI dataset as container are acceptable

Reply:
Yeap! checking through. actually corpuserkey is just having inconsistent json ordering. 

I have ensured that the new feature will have no impact on existing user as much as possible.

Reply:
description and custom properties update are new. So are subtypes. The rest remain the same. U can open in full view and check.